app/controllers/datasets_controller.rb

```diff
@@ -154,6 +154,9 @@ def facet_results(request, response)
 
     keyword_facets.sort_by! {|facet| facet['term']}
     results.insert(1, {name: "Keywords", param: nil, values: selected_keywords + keyword_facets})
+    # EDSC-698 Dropping sensor as a facet.
+    # FIXME: This may need retouch once CMR drops the sensor facet.
+    results.reject!{|facet| !facet.nil? && facet[:name] == 'Sensor'}
```

Comment:
Do the compact before this and get rid of the `.nil?` check. This and the following line can be combined into just:

    results.compact.reject { |facet| facet[:name] == 'Sensor' }